scripts/inventory/packet_net.py

```diff
@@ -346,9 +346,9 @@ def add_device(self, device, project):
 
         # Inventory: Group by OS
         if self.group_by_operating_system:
-            self.push(self.inventory, device.operating_system['slug'], dest)
+            self.push(self.inventory, device.operating_system.__dict__['slug'], dest)
```

Comment:
I don't know this inventory script at all, and neither do I know the packet-python library (https://pypi.org/project/packet-python/, https://github.com/packethost/packet-python). I've looked a bit at it's source. `device` is an instance of `Device` (https://github.com/packethost/packet-python/blob/927684bda1b5092ab14327ac517f5dcad2e7cafd/packet/Device.py#L7), and `device.operating_system` an instance of OperatingSystem (https://github.com/packethost/packet-python/blob/927684bda1b5092ab14327ac517f5dcad2e7cafd/packet/OperatingSystem.py#L5). It looks like the correct form to access this is `device.operating_system.slug`. `__dict__` is how objects are implemented, and not something that should be accessed directly (if possible). I'm not sure which version of the library the author of this script has been using, but the library in https://github.com/packethost/packet-python/ never seemed to have supported array subscription.

So a better change would be:
```suggestion
            self.push(self.inventory, device.operating_system.slug, dest)
```
(Probably same for the others.)

Reply:
Sounds good to me, and that is really cool that you can make suggestions like that! I didn't know you could do that, but yep I just finished testing on my end to make sure it worked. Thank you for working with me on this, as I am very new to the project :upside_down_face:

Reply:
You're welcome! Could you please remove the mention (`@`) from the commit messages? (Once the bot runs again, it will complain about it.)